screens/ReportGenerationScreen.tsx

```diff
@@ -0,0 +1,177 @@
+import { useState } from 'react';
+import { ScrollView, StyleSheet, View } from 'react-native';
+import { Button, Caption, Headline, Icon } from 'react-native-paper';
+import { DatePickerInput } from 'react-native-paper-dates';
+import { NativeStackScreenProps } from '@react-navigation/native-stack';
+import { Appbar } from '../components/Appbar';
+import { FooterScreens, FooterView } from '../components/Footer';
+import { FullScreenActivityIndicator } from '../components/FullScreenActivityIndicator';
+import { useNucaTheme as useTheme } from '../hooks/useNucaTheme';
+import { RootStackParamList } from '../types';
+import { isSmallScreen } from '../utils/helperFunc';
+
+const getStyles = (theme: NucaCustomTheme) =>
+  StyleSheet.create({
+    saveButton: {
+      height: 64,
+      maxWidth: 340,
+      minWidth: isSmallScreen() ? 64 : 340,
+    },
+    saveButtonContent: {
+      height: 64,
+      alignItems: 'center',
+      justifyContent: 'center',
+      flexDirection: 'row-reverse',
+    },
+    saveButtonLabel: {
+      fontSize: 18,
+      marginStart: 8,
+      textAlignVertical: 'bottom',
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.background,
+    },
+    loadingContainer: {
+      flex: 1,
+      justifyContent: 'center',
+      alignItems: 'center',
+    },
+    container: {
+      backgroundColor: theme.colors.background,
+      flex: 1,
+    },
+    scrollView: {
+      width: '100%',
+      height: '1000%',
+    },
+    contentContainer: {
+      padding: 20,
+      width: isSmallScreen() ? '100%' : '85%',
+    },
+    screenTitleContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+    },
+    screenTitleIcon: {
+      fontSize: 28,
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.text,
+    },
+    screenTitleLabel: {
+      fontSize: 18,
+      marginStart: 8,
+      textAlignVertical: 'bottom',
+      fontFamily: 'Nunito_700Bold',
+      color: theme.colors.text,
+    },
+    pickerContainer: {
+      flexDirection: 'row',
+      alignItems: 'center',
+      flexWrap: 'wrap',
+    },
+    datePickerContainer: {
+      width: '50%',
+      flexDirection: 'column',
+      justifyContent: 'flex-start',
+      alignItems: 'flex-start',
+      padding: 4,
+    },
+    textInputTitle: {
+      color: theme.colors.text,
+      fontSize: 12,
+      marginTop: 20,
+      fontFamily: 'Nunito_700Bold',
+      textTransform: 'uppercase',
+    },
+    titleRow: {
+      flexDirection: 'row',
+      alignItems: 'baseline',
+      paddingBottom: 12,
+    },
+  });
+
+export const ReportGenerationScreen = (
+  _params: NativeStackScreenProps<RootStackParamList, 'ReportGeneration'>
+) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const [isInProgress, _setIsInProgress] = useState(false);
+
+  return (
+    <>
+      <Appbar forDetailScreen />
+      <View style={styles.container}>
+        <ScrollView style={styles.scrollView}>
```

Comment:
I noticed in the screens that the background image extends lower than the white background... can you fix this?

Reply:
Disabling the `ScrollView` solves the issue on mobile, but not on web. I tried to remove the `ScrollView`, but the contents of the screen are not displayed as expected (date picker captions are displayed behind the date picker elements themselves).
Other than that, I tried using the `transform: translate` prop, but that doesn't seem to work at all. Perhaps we could note down this bug as a separate task?

Reply:
I managed to fix this issue anyways &rarr; https://github.com/crafting-software/nuca-mobile/pull/75#discussion_r1629088627